### PR TITLE
add example to RegExpPart1.py and create new file as special_method.py

### PR DESCRIPTION
**Pull request by ZijuanZhang**

### CODE REVIEW

File: text_processing/RegExpPart1.py [hunk 1]
```diff
@@ -53,9 +53,6 @@
 x = re.match("cat", "A cat and a rat can't be friend.")
 print(x)
 
-# will match from start of string
```
Comment (JesseZhuang) [inline text_processing/RegExpPart1.py:56]: why did you delete these 3 lines I added?

Comment (ZijuanZhang) [reply text_processing/RegExpPart1.py:56]: Not pay attention to this. I changed the code with last time's. And you added them during the meeting, maybe this is the reason.

Comment (JesseZhuang) [reply text_processing/RegExpPart1.py:56]: So please add them back.
